About iterations (#1025)
hashicorp/design-system: website/docs/about/support.md

---
title: Support
order: 103
---

!!! Info

Many resources and links on this page are only available to HashiCorp employees.
!!!

> We're happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help if you get stuck.

Reviewer: ```suggestion
We’re happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help.
```
Including @alex-ju 's suggestion but also removing the "if you get stuck" as you don't need to be "stuck" to seek out help from us.

## Live support

> Join the [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) Slack channel to ask questions, request and provide feedback, or start a conversation with our team. As a globally-distributed team, there may be a slight delay in answering some questions, but we're committed to responding in a timely manner.

Reviewer 2: ```suggestion
Join the [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) Slack channel to ask questions, request and provide feedback, or start a conversation with our team. As a globally-distributed team, there may be a slight delay in answering some questions, but we’re committed to responding in a timely manner.
```
